src/CarModel.php

```diff
@@ -99,4 +99,39 @@ public function deleteCarById(string $id): bool
             return false;
         }
     }
+
+    public function editCarDetails(Car $car): bool
+    {
+        $id = $this->getIdFromName($car->name);
```

Comment:
Tried doing this for an hour and its a nightmare to change now and besides I think you would still be doing the same number of get id calls since you cant know the id of the car when you create it anyways... Something for me to think about when creating these type of classes in the future

Reply:
actually never mind I soled it in like 5 mins today